alphagov/gsp-terraform-ignition
# Bump Concourse version

## What

Concourse 5.2 is out. :tada:

It comes with the shiny new `pipeline-operator` role which will be
useful when supporting our users.

It will allow them to log into the UI and hit the big shiny `+` button.

## How to review

- Sanity check
- Deploy to cluster from this branch
- See it in action
